src/testing_tools.c

```diff
@@ -11,7 +11,7 @@ bool fail_xmlAddChild = false;
 bool fail_xmlXPathNewContext = false;
```

Comment:
Camel case and underline, but I guess this is because of the xml lib? (there are others below like `xmlChar`, `xmlVersion`, etc).

Reply:
I think in this case is a good idea to leave them as they're for readability reasons.